index.html

```diff
@@ -155,6 +155,27 @@ <h1 class="section-header">How to Use</h1>
                 </div>
               </div>
               <br>
+              <div class="text">
+                To produce disabled buttons, add disabled attribute to the button as shown below.
```

Comment:
Change text to "To make a button disabled, ..."

Reply:
requested change is added! Please review! I added example for both button and anchor tags. let me know if thats fine!